### PR TITLE
Save ReleaseDate as Pref instead of Cookie

### PR DESCRIPTION
**Pull request by Langleu**
needs https://github.com/schul-cloud/schulcloud-server/pull/309

### CODE REVIEW

**Comment by janrenz** [comment]
Locks great. But maybe we can even go one step further: We could read both preferences and release date on the server and then only trigger the client to show release info once. This would avoid the call to /help/releases on every page..

**Comment by Langleu** [comment]
if we had stuck to React and websockets this would have been an easy one.
I'll look what I can do

**Comment by janrenz** [comment]
also it would be ok to do this only as part of the login action

**Comment by Langleu** [comment]
sure thing. I'll try some stuff out.

**Comment by Langleu** [comment]
@janrenz changed some stuff and it now would use websockets.
The User logs in and the server checks whether the user has seen the newest release, if not it triggers an event via websocket.
Will probably still move it to the dashboard, as it's overkill to check it every time.

**Comment by janrenz** [comment]
plus one for moving it to dashboard

**Comment by Langleu** [comment]
in a frontend script?

**Comment by janrenz** [comment]
should be stored in a env and then somehow passed to the script.. otherwise this wouldn't run in production or am I overseeing something?

**Comment by Langleu** [comment]
changed it, env variable will be to much of a mess to get it into the frontend.

File: static/scripts/socket.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 $(document).ready(function () {
-    let socket = io('http://localhost:3030');
+    let url = window.location.href.includes('localhost') ? 'http://localhost:3030' : 'https://schul-cloud.org:8080';
```
Comment (schurigh) [inline static/scripts/socket.js]: I'm running http://127.0.0.1:3100/ locally ;)

Comment (Langleu) [reply static/scripts/socket.js]: switched the cases, shouldn't matter anymore.

**Comment by Langleu** [comment]
allet klar, mach ich dann beim nächsten Mal.

**Comment by Langleu** [comment]
@janrenz done.

File: helpers/handlebars/middleware.js [hunk 2]
```diff
@@ -33,6 +33,7 @@ const makeActive = (items, currentUrl) => {
 
 module.exports = (req, res, next) => {
     res.locals.themeTitle = process.env.SC_NAV_TITLE || 'Schul-Cloud';
+    res.locals.backendUrl = process.env.BACKEND_URL || 'http://localhost:3030';
```
Comment (schurigh) [inline helpers/handlebars/middleware.js]: Socket.io won't work if you test on http://127.0.0.1 instead of localhost but that might be unusual

**Comment by Langleu** [comment]
changed it to the public backend url as the normal backend url is used for docker referencing meaning server:3030 aint gonna work for any connections

**Comment by janrenz** [comment]
on mobile we dont show the release info pop up

**Pull request merged**